src/tools/pub/pubspec.md

```diff
@@ -269,6 +274,43 @@ For more information, see
 [pub global](/tools/pub/cmd/pub-global#running-a-script-from-your-path).
 
 
+### Platforms
```

Comment:
I like this overall, but I think we need the "For Flutter packages you can use the [Flutter platforms tag][]." part from https://github.com/dart-lang/site-www/pull/3794/files